spec/introduction.html

```diff
@@ -38,4 +38,8 @@ <h1>Introduction</h1>
     Leo Balter, Valerie Young, Isaac Durazo<br>
     ECMA-402, 7<sup>th</sup> Edition Project Editors
   </p>
+  <p>
+    Leo Balter<br>
+    ECMA-402, 8<sup>th</sup> Edition Project Editor
+  </p>
```

Comment:
cc @spectranaut @isaacdurazo, please let me know if you want to continue the editorship for the 8th Edition (2021) work. I'd be happy to change this.